docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>SMC (with a Metropolis kernel) is not free of problems <strong>though. As</strong> it relies on Metropolis<strong>,</strong> it will deteriorate <s>as</s> <strong>if</strong> the dimensionality of the problem increases and/or if the geometry of the posterior is weird<strong>,</strong> as in hierarchical models.</li><li>Let's make SMC fail<s>s</s> spectacularly</li><li>Is the SMC used in PyMC3 based on NUTS or Metropolis BTW? Could be nice to add this precision</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>This wasn't addressed in the latest commit</p>